README.md

```diff
@@ -21,6 +21,51 @@ _**Note**: below listed features are still under design and review phase and hen
 * New range functions
 * Freeze rows/columns.
 
+## Upcoming Excel 1.8 Event Release Features
+
+Below is the early thought of Event included in Excel JavaScript APIs. Further details about event argument design and sample code will be posted later. 
+
+1.	Data change event for Worksheet, Table with co-auth support
+
+    This type of event is triggered when there is data change happened, for example, cells value change on the grid.
+
+    |Object|Event| Description |
+    |:----|:----|:----|
+    | Worksheet    | onDataChannged | Occurs when data changed on specific worksheet |
+    | Table        | onDataChannged | Occurs when data changed on specific table |
+    | TableCollection      | onDataChannged | Occurs when data changed on any table in a workbook, or a worksheet |
+
+2.	Selection change event for Worksheet, and Table.
+
+    This type of event is triggered when user changes the selection, or extend the selected area.
+
+    |Object|Event| Description |
+    |:----|:----|:----|
+    |Worksheet| onSelectionChanged | Occurs when selection changed on specific worksheet |
+    |Table| onSelectionChanged | Occurs when selection changed on specific table |
+
+
+3.	Activate and Deactivate events for Worksheet.
+
+    This type of event is triggered when worksheet is activated, for example, user switches the worksheet.
```

Comment:
Revise: This type of event is triggered when a worksheet is activated; for example, when a user switches the worksheet.